runtime/doc/repeat.txt

```diff
@@ -516,16 +516,26 @@ You would now have these files under ~/.local/share/nvim/site:
 	pack/foo/start/foobar/syntax/some.vim
 	pack/foo/opt/foodebug/plugin/debugger.vim
 
-When Vim starts up, after processing your .vimrc, it scans all directories in
-'packpath' for plugins under the "pack/*/start" directory.  First all those
-directories are added to 'runtimepath'.  Then all the plugins are loaded.
-See |packload-two-steps| for how these two steps can be useful.
+							*runtime-search-path*
+When runtime files are searched for, first all paths in 'runtimepath' are
+searched, then all "pack/*/start/*" dirs are searched. However, package entries
+are not visible in `:set rtp` or `echo &rtp`, as the final concatenated path
+would be too long and get truncated. To list all used directories, use
```

Comment:
we don't really need to mention this reason, which is a pretty lame reason anyway (and doesn't capture our full reasoning?).

```suggestion
are not visible in `:set rtp` or `echo &rtp`. To list all used directories, use
```